tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -3152,7 +3174,8 @@ Status TfTensorToTrtWeights(const Tensor& tensor, TrtWeightStore* weight_store,
   // this.
   DataType converted_dtype = dtype;
   if (dtype == DataType::DT_INT8 || dtype == DataType::DT_UINT8 ||
-      dtype == DataType::DT_INT16 || dtype == DataType::DT_UINT16) {
+      dtype == DataType::DT_INT16 || dtype == DataType::DT_UINT16 ||
+      dtype == DataType::DT_INT64) {
```

Comment:
Just curious: do we want to consider uint64 and uint32 as well?

Reply:
@trevor-m Do we want to support these types?

Reply:
Sure, we might as well